src/main/java/domain/command/CommandParser.java

```diff
@@ -0,0 +1,29 @@
+package domain.command;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import view.CommandType;
+
+public class CommandParser {
+    public static final Map<CommandType, Function<List<String>, Command>> commandByChessCommand;
```

Comment:
`Function<List<String>, Command>` 부분은 가독성이 떨어지는 것 같아요. 
해당 역할을 함수형 인터페이스로 따로 정의해 주는 것도 좋을 것 같아요
```suggestion
    public static final Map<CommandType, CommandGenerator> commandByChessCommand;
```